ui-components/urldialog.md

```diff
@@ -7,91 +7,90 @@ keywords: dialog urldialog api
 ---
 ## Overview
 
-A URL dialog is a special TinyMCE UI component which loads an external web page inside a dialog (sometimes referred to as `modals`). This differs from the regular dialogs that use supported components to render an interactive dialog inside the application. URL dialogs are useful for very complex use cases, where the supported dialog components cannot be used — for example, a custom file manager that is loaded inside a TinyMCE dialog.
+A URL dialog is a special TinyMCE UI component which loads an external web page inside a dialog (sometimes referred to as `modals`). URL dialogs are useful for very complex use cases, where the supported components for TinyMCE's standard dialogs cannot be used. For example, a custom file manager that is loaded inside a TinyMCE dialog would probably require a URL dialog.
 
-The most basic configuration structure is:
+> Note: [Standard TinyMCE dialogs]({{site.baseurl}}/ui-components/dialog/) should suffice for most use cases, and may be simpler to configure.
+
+### Basic example
+
+The configuration for a basic URL dialog might look like this:
 
 ```js
-const urlDialogConfig = {
+tinymce.activeEditor.windowManager.openUrl({
    title: 'Just a title',
    url: 'http://www.tiny.cloud/example.html'
-}
+});
 ```
 
 ## URL dialog configuration
 
-A URL Dialog configuration has two main parts:
+A URL Dialog configuration has three main parts to match the three main parts of the dialog's UI:
+
+* **Title:** The title of the dialog. This will display in the header of the dialog.
 
-* **Title** This is the title of a dialog.
+* **URL:** The URL of the external page to load inside the dialog.
 
-* **URL** The external pages URL to load inside the dialog.
+* **Buttons: optional -** An array of [footer buttons](#footerbuttons) that are displayed in the dialog's footer.
 
-### Config options
+### Configuration options
 
 | Name    | Value  | Requirement | Description |
 | ------- | ------ | ----------- | ----------- |
 | title   | string | required    | The title of the dialog. |
 | url     | string | required    | The URL to the external page to load. |
 | width   | number | optional    | The width of the dialog in pixels. |
 | height  | number | optional    | The height of the dialog in pixels. |
-| buttons | UrlDialogButton[] | optional | An optional array of button configurations to render in the footer. See [Footer components](#footercomponents) configuration. |
-| onAction | (api) => void | optional | Function invoked when a custom button is clicked. |
-| onCancel | (api) => void | optional | Function invoked when the dialog is cancelled. |
-| onClose | () => void | optional | Function invoked when the dialog has been closed. |
-| onMessage | (api, data) => void | optional | Function invoked when a message is received from the external page. |
-
-### Footer components
+| buttons | FooterButton[] | optional | An optional array of [footer buttons](#footerbuttons) to render in the footer of the dialog. |
```

Comment:
It feels like we're repeating `footer` too much here maybe? Maybe instead change the first reference to `button configurations` or even just simply `buttons`?